src/test/cpp/test/Wil.cpp

```diff
@@ -0,0 +1,53 @@
+#include "pch.h"
+#include "winrt/Windows.Foundation.h"
+
+using namespace winrt;
+using namespace Windows::Foundation;
+
+namespace
+{
+    // Some custom exception type unknown to C++/WinRT
+    struct WilException
```

Comment:
I know WIL is the prime beneficiary, but a more general name would be clearer (e.g., CustomException)

Reply:
Sure. 😉